crates/driver/src/infra/observe/mod.rs

```diff
@@ -182,15 +182,22 @@ pub fn settled(solver: &solver::Name, result: &Result<competition::Settled, comp
 }
 
 /// Observe the result of solving an auction.
-pub fn solved(solver: &solver::Name, result: &Result<Solved, competition::Error>) {
+pub fn solved(solver: &solver::Name, result: &Result<Option<Solved>, competition::Error>) {
     match result {
-        Ok(solved) => {
+        Ok(Some(solved)) => {
             tracing::info!(?solved, "solved auction");
             metrics::get()
                 .solutions
                 .with_label_values(&[solver.as_str(), "Success"])
                 .inc();
         }
+        Ok(None) => {
+            tracing::info!("no solution found");
```

Comment:
not even sure this is worth an info log (maybe debug is fine)

Reply:
Good point.